01-introduction.Rmd

```diff
@@ -72,8 +74,54 @@ In this book, you will come across terms like `FeatureIDs` and `SampleIDs`.
 `colnames` are encouraged to be used, since they relate to actual objects we 
 work with.
 
+
+
 ## Loading experimental microbiome data
 
+
+### Importing data 
+
+
+TODO: data importer from standard CSV/TSV/tree files directly into `TSE`?
+
+Alternative input file formats: data from QIIME2 file formats can be
+directly imported in `TreeSummarizedExperiment` format. See
+`help(mia::makeTreeSummarizedExperimentFromqiime2)` for examples.
```

Comment:
Also phyloseq, biom and DADA2 are supported. My guess would be that they should be mentioned here as well